src/embedWebFonts.ts

```diff
@@ -34,55 +37,86 @@ function fetchCSS(url: string, window: Window): Promise<void | Metadata> {
   return deferred
 }
 
+function selectorsCompletelyAppliesToRule(
+  selectors: string[],
+  rule: CSSRule,
+): boolean {
+  if (rule.type === CSSRule.STYLE_RULE) {
+    const styleRule = rule as CSSStyleRule
+    const styleRuleSelectors = styleRule.selectorText.split(',')
+
+    return styleRuleSelectors.every((ruleSelector) => {
+      return selectors.some((selector) => ruleSelector.includes(selector))
```

Comment:
yes, I'm only doing the minimum work here

Reply:
The wildcard should fail according to this implementation. For instance, if the CSS rule is `p * ` and the selector provided is `p div`, it won't match because it's just comparing strings. I am ok with that. Anyway, I would suggest mention these limitation in the ts file - https://github.com/OutSystems/html-to-image/blob/d768bf5d26c817198d6481d8a658d8db2cefad44/src/options.ts#L103

Reply:
done